src/Compute/Compute/help/New-AzVM.md

```diff
@@ -1154,7 +1154,8 @@ Accept wildcard characters: False
 ```
 
 ### -Zone
-Specifies the zone list of the virtual machine.
+Specifies the zone of the virtual machine. Although it takes in an array of zones, virtual machines do not support multiple availability zones.
+The allowed value depend on the capabilities of the region. Allowed value will normally be 1,2, or 3. More information on [Azure availability zones](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-overview#availability-zones).
 
 ```yaml
 Type: System.String[]
```

Comment:
>System.String[] [](http://example.com/codeflow?start=6&length=15)

Oh... this answer one my question above. The param is indeed declared as an array of strings. Probably testament of the REST-y nature of the underlying libraries.

Still, I think that PowerShell is smart enough to map 1 to "1" when you type.

Reply:
it does but I want to show correct notation. that it is a string not an int.